### ISSUE TITLE
Less verbose logs on function invocation

### ISSUE THREAD

**Issue by pragnagopa** [Azure/azure-functions-core-tools#2143]
Part of #1131 

This issue tracks following:

(copied from https://github.com/Azure/azure-functions-core-tools/issues/1131#issuecomment-643655379)

### Function invocation

* Remove the HTTP response JSON output
* Would be great if we could de-emphasize the executing/executed messages to make it easy to pick out logs from user code

```
Executing 'Functions.HttpTrigger' (Reason='This function was programmatically called via the host APIs.', Id=b7b32e4f-4c9f-4a48-b160-5d445644f182)
JavaScript HTTP trigger function processed a request.
Executed 'Functions.HttpTrigger' (Succeeded, Id=b7b32e4f-4c9f-4a48-b160-5d445644f182)
```

**Comment by pragnagopa** [comment]
@anthonychu - Setting `LogLevel=Warning` for System Logs and `LogLevel=Information` for user logs, shows following on invocation

```
JavaScript HTTP trigger function processed a request.
```

I think this achieves the default experience.

If user specifies default log level in host.json such as the following:

```
"Logging": {
      "LogLevel": {
		"Default": "Information"
        }
   }
```

this would override default log levels set in the core tools.

**Comment by anthonychu** [comment]
@pragnagopa Yes that's a good point. We didn't specify behavior when the customer specifies the log levels. I agree that the user's config should win.

**Comment by pragnagopa** [comment]
Addressed in referenced PR #2139
